src/test/store/profile-view.test.js

```diff
@@ -945,14 +945,13 @@ describe('snapshots of selectors/profile', function() {
       ['D', 3, null],
       ['E', 4, null],
       ['F', 5, null],
-      getNetworkMarker(6, 6),
-      getNetworkMarker(7, 7),
+      ...getNetworkMarkers(6, 6),
+      ...getNetworkMarkers(7, 7),
     ]);
     profile.threads.push(markersThread);
     const { getState, dispatch } = storeWithProfile(profile);
     samplesThread.name = 'Thread with samples';
     markersThread.name = 'Thread with markers';
-    samplesThread.markers = markersThread.markers;
```

Comment:
as discussed on Slack, the "markers" raw table holds references to the string table, so this was incorrect.

Then I changed the marker-related selectors below to refer to the markersThread instead of the samplesThread.